src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -251,19 +265,19 @@ export class HaAutomationEditor extends LitElement {
   protected updated(changedProps: PropertyValues): void {
     super.updated(changedProps);
 
-    const oldAutomation = changedProps.get("automation") as AutomationEntity;
+    const oldAutomationId = changedProps.get("automationId");
     if (
-      changedProps.has("automation") &&
-      this.automation &&
+      changedProps.has("automationId") &&
+      this.automationId &&
       this.hass &&
       // Only refresh config if we picked a new automation. If same ID, don't fetch it.
-      (!oldAutomation ||
-        oldAutomation.attributes.id !== this.automation.attributes.id)
+      oldAutomationId !== this.automationId
     ) {
+      this._setEntityId();
       this.hass
```

Comment:
We should probably extract this into an async method and put the API call in `data/automation`, but maybe not for this PR.

Reply:
It is because of the debounce of automations, a new created automation is not in automations instantly

Reply:
Oh it is just about the call, yeah we should